crates/bdk/src/wallet/coin_selection.rs

```diff
@@ -836,7 +836,7 @@ mod test {
         let drain_script = ScriptBuf::default();
         let target_amount = 250_000 + FEE_AMOUNT;
 
-        let result = LargestFirstCoinSelection::default()
+        let result = LargestFirstCoinSelection
```

Comment:
These changes were as a result of running clippy. If you run clippy on master you will get recommendations to remove them. Since in rust tuple struct `LargestFirstCoinSelection` you dont need to call `::default()` to initialize it

Reply:
Generally it's better to handle fixing clippy changes in a separate commit or PR to make the review easier. Also a nit on commit messages, instead of `BREAKING CHANGE:` I'd use the notation `refactor!(<module name>): ...`. See: https://www.conventionalcommits.org/en/v1.0.0/

And as @LLFourn mentioned we may end up incorporating this rename into another PR but if you have time it wouldn't hurt to fix this one up in case we can use it.